week-1-practise/level-1/10-function-output-3.js

```diff
@@ -1,5 +1,8 @@
 function concatenate(firstWord, secondWord, thirdWord) {
     // Write the body of this function to concatenate three words together
+    let firstConcat = firstWord.concat(secondWord);
+    let finalConcat = firstConcat.concat(thirdWord);
+    return finalConcat;
```

Comment:
This works well. In this case though, maybe we can do this without having two different variables declared. Can you think of a way?